### PR TITLE
Feature/23 fast register,time shifting, date shifting

### PR DESCRIPTION
**Pull request by VitaliKhileuski**

### CODE REVIEW

File: hotel-reservation-system/src/App.js [hunk 1]
```diff
@@ -21,7 +29,7 @@ export default function App() {
         localStorage.setItem("refreshToken", tokenRefreshResponse.data[1]);
         failedRequest.response.config.headers["Authorization"] =
           "Bearer " + tokenRefreshResponse.data[0];
-        FillStorage(tokenRefreshResponse.data[0], dispatch);
+        FillStorage(tokenRefreshResponse.data[0]);
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/App.js:32]: Please name functions in camelCase

File: hotel-reservation-system/src/components/Authorization/FastRegister.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Typography component="h1" variant="h5">
+          Fast Register
+        </Typography>
+        <Formik
+          initialValues={initialValues}
+          onSubmit={(values) => {
+            if (ValidateEmail(email.trim()) && emailErrorLabel === "")
+              onSubmit(values);
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/components/Authorization/FastRegister.js:99]: Please move this to the function handleSubmit

File: hotel-reservation-system/src/components/Hotel/AddHotelForm.js [hunk 3]
```diff
@@ -109,18 +122,50 @@ export default function AddHotelForm({
...
+
   return (
     <>
       <Container component="main" maxWidth="xs" className={classes.root}>
         <CssBaseline />
         <div className={classes.paper}>
           <Formik
             initialValues={initialValues}
-            onSubmit={onSubmit}
+            onSubmit={(values) => {
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/components/Hotel/AddHotelForm.js:156]: Please move to the separate function

File: hotel-reservation-system/src/components/Hotel/HotelTable.js [hunk 4]
```diff
@@ -245,7 +245,14 @@ export default function HotelTable() {
       setCurrentAscending("desc");
       ascending = "desc";
     }
-    loadHotels(undefined, undefined, undefined, sortField, ascending);
+    loadHotels(
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/components/Hotel/HotelTable.js:248]: Are you sure that function is correct if you need to pass so many undefined values?

File: hotel-reservation-system/src/components/Room/RoomTable.js [hunk 5]
```diff
@@ -116,7 +125,7 @@ export default function RoomTable({ hotelId }) {
       setCurrentAscending("desc");
       ascending = "desc";
     }
-    loadRooms(undefined, undefined, sortField, ascending);
+    loadRooms(undefined, undefined, undefined, sortField, ascending);
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/components/Room/RoomTable.js:128]: same

File: hotel-reservation-system/src/components/Service/ServiceTable.js [hunk 6]
```diff
@@ -205,71 +194,96 @@ export default function ServiceTable({ hotelId, serviceList }) {
...
-                  >
-                    Payment
-                  </TableSortLabel>
-                </TableCell>
                 {!!hotelId ? (
-                  ""
+                  <>
+                    <TableCell align="right" style={{ minWidth: 170 }}>
+                      <TableSortLabel
+                        active={currentSortField === "Name" ? true : false}
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/components/Service/ServiceTable.js:201]: active={currentSortField === "Name"}

File: hotel-reservation-system/src/components/Service/ServiceTable.js [hunk 7]
```diff
@@ -205,71 +194,96 @@ export default function ServiceTable({ hotelId, serviceList }) {
...
+                        direction={currentAscending}
+                        onClick={() => orderBy("Name")}
+                      >
+                        Name
+                      </TableSortLabel>
+                    </TableCell>
+                    <TableCell align="right" style={{ minWidth: 100 }}>
+                      <TableSortLabel
+                        active={currentSortField === "Payment" ? true : false}
```
Comment (PolinaMikhno) [inline hotel-reservation-system/src/components/Service/ServiceTable.js:210]: same

File: hotel-reservation-system/src/components/Service/ServiceTable.js (hunk 6 above)
Comment (VitaliKhileuski) [reply hotel-reservation-system/src/components/Service/ServiceTable.js:201]: deleted

File: hotel-reservation-system/src/components/Service/ServiceTable.js (hunk 7 above)
Comment (VitaliKhileuski) [reply hotel-reservation-system/src/components/Service/ServiceTable.js:210]: deleted

File: hotel-reservation-system/src/components/Authorization/FastRegister.js (hunk 2 above)
Comment (VitaliKhileuski) [reply hotel-reservation-system/src/components/Authorization/FastRegister.js:99]: moved

File: hotel-reservation-system/src/App.js (hunk 1 above)
Comment (VitaliKhileuski) [reply hotel-reservation-system/src/App.js:32]: renamed

File: hotel-reservation-system/src/components/Hotel/AddHotelForm.js (hunk 3 above)
Comment (VitaliKhileuski) [reply hotel-reservation-system/src/components/Hotel/AddHotelForm.js:156]: moved

**Pull request merged**